src/main/java/org/lightcouch/View.java

```diff
@@ -428,7 +430,7 @@ public boolean queryForBoolean() {
      * @param key The key value, accepts a single value or multiple values for complex keys.
      */
     public View key(Object... key) {
-        this.key = getKeyAsJson(key);
+        this.key = getKeyAsJsonElement(key);
         uriBuilder.query("key", this.key);
```

Comment:
After the change to using JsonElement rather than String, to fix the bug with complex keys, you are implicitly relying on JsonElement's `toString` method returning a valid JSON string here for us as part of the URL, which isn't part of the `toString` method's contract. 

Use `Gson.toJson` to guarantee you are sending in valid JSON in the URL. 

The same goes for `startKey` and `endKey`, below.